src/components/Tag/Tag.jsx

```diff
@@ -0,0 +1,96 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+
+const boundClassNames = lucidClassNames.bind('&-Tag');
+const {
+	bool,
+	func,
+	node,
+	object,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["navigation"]}
+ *
+ * Tags are keywords used to label, organize and categorize content that can be
+ * mapped to multiple categories.
+ */
+const Tag = React.createClass({
+	propTypes: {
+		/**
+		 * The representation of the keyword that this instance represents -- can be
+		 * any renderable content.
+		 */
+		children: node,
+
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Indicates whether the component should include an "X" that the user can
+		 * click on to close/hide/remove it.
+		 */
+		isCloseable: bool,
+
+		/**
+		 * Called when the user clicks on the "X" (if included).
+		 */
+		onClose: func,
+
+		/**
+		 * Passed through to the root element.
+		 */
+		style: object
+	},
+
+	getDefaultProps() {
+		return {
+			isCloseable: false,
```

Comment:
I'm iffy on that. If we look at it just from an internal usage perspective, then yeah it would make sense to make this true by default. However, I'd argue that the most common scenario "in the wild" where you'd see tags is on pages where you don't have the ability to remove them. In that sense I think the default would be not closeable...